light-client/examples/light_client.rs

```diff
@@ -85,13 +85,7 @@ fn sync_cmd(opts: SyncOpts) {
         light_store.insert(trusted_state, VerifiedStatus::Verified);
     }
 
-    let peers = Peers {
-        primary,
-        witnesses: Vec::new(),
-    };
-
-    let state = State {
-        peers,
+    let mut state = State {
```

Comment:
It seems like we keep leaning on `use ...::prelude::*` which obscures the package structure and makes it pretty difficult to figure out where structs are coming from. From a comprehension perspective it may be better to just import the structs we need from their locations.

Reply:
Agreed, I plan on removing it asap.